docs/changes/io.fits/12544.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Add a ``mask_invalid`` option to ``Table.read`` to deactivate the masking of
+NaNs in float columns and empty strings in string columns.
```

Comment:
Add something to explain why this is useful, like `This option is necessary to allow effective use of memory-mapped reading with memmap=True`.

Reply:
Done, thanks.